readme.md

```diff
@@ -28,6 +28,23 @@ If you're a developer, and want to get started and [contribute](#Contributing) t
 5) Run `CRender.exe` and enjoy!
 
 ***
-## Contributing
+## Building
+
+### Linux
+
+1) To build this project you will require [Intel Rendering Toolkit](https://software.intel.com/content/www/us/en/develop/tools/oneapi/rendering-toolkit/download.html) (online version), CMake, and `libxi-dev build-essential xorg libglfw3 libglfw3-dev libxinerama-dev libxcursor-dev` (these are the packages for Debian/Ubuntu)
+
+2) You will need to run `source /opt/intel/oneapi/setvars.sh` so it sets the global vars needed for building. If this doesn't work please look for this file, and source that file in the same terminal window you compile in.
```

Comment:
You also need onetbb from Intel's multi-threading kit.

Reply:
Yup, I added those in withing the latest commit.